Dockerfile

```diff
@@ -1,8 +1,18 @@
-FROM ubuntu:precise
+FROM ubuntu:trusty
+MAINTAINER Jason Gegere <jason@htmlgraphic.com>
+
+ADD ./bin/ /usr/local/sbin
+RUN chmod 755 /usr/local/sbin/*
 RUN echo deb http://archive.ubuntu.com/ubuntu/ precise main universe > /etc/apt/sources.list.d/precise.list
```

Comment:
This is used to test more of the current repo. It might be possible to remove it. To setup tests on circleCI.com would be great!